docs/docs/examples/styling.mdx

```diff
@@ -68,74 +96,14 @@ import 'react-tooltip/dist/react-tooltip.css'
   <TooltipAnchor id="custom-styles" data-tooltip-content="hello world!">
     ◕‿‿◕
   </TooltipAnchor>
-  <Tooltip anchorId="custom-styles" className="example" classNameArrow="example-arrow" />
+  <Tooltip anchorId="custom-styles" className="example" />
 </div>
 
 #### Explanation
 
 In this example, we are adding an extra level to the CSS classes, the following styles are the default one of Tooltip component when we write this docs:
 
-```css
-.tooltip {
-  visibility: hidden;
-  width: max-content;
-  position: absolute;
-  top: 0;
-  left: 0;
-  padding: 8px 16px;
-  border-radius: 3px;
-  font-size: 90%;
-  pointer-events: none;
-  opacity: 0;
-  transition: opacity 0.3s ease-out;
-  will-change: opacity, visibility;
-}
-
-.arrow {
-  position: absolute;
-  background: var(--rt-color-dark);
-  width: 8px;
-  height: 8px;
-  transform: rotate(45deg);
-}
-
-/** Types variant **/
-.dark,
-.dark .arrow {
-  background: var(--rt-color-dark);
-  color: var(--rt-color-white);
-}
-
-.light,
-.light .arrow {
-  background-color: var(--rt-color-white);
-  color: var(--rt-color-dark);
-}
-
-.success,
-.success .arrow {
-  background-color: var(--rt-color-success);
-  color: var(--rt-color-white);
-}
-
-.warning,
-.warning .arrow {
-  background-color: var(--rt-color-warning);
-  color: var(--rt-color-white);
-}
-
-.error,
-.error .arrow {
-  background-color: var(--rt-color-error);
-  color: var(--rt-color-white);
-}
-
-.info,
-.info .arrow {
-  background-color: var(--rt-color-info);
-  color: var(--rt-color-white);
-}
-```
+<CodeBlock language="css">{TooltipStyles}</CodeBlock>
```

Comment:
We need to validate the docs inline in this part when the PR gets merged to master.

I'm not sure if the import will work as expected because the root folder to be deployed in Digital Ocean is `docs`, not the repository folder

Reply:
not sure how this works, never used `.mdx`

Reply:
no problem, this is most a reminder to me check this part of docs when we deploy this PR